File: GATE ONE/DAYSEVEN/JAVA/DIARY/DiaryEntry.java
import java.time.LocalDateTime;

public class DiaryEntry {
    private int id;
    private String title;
    private String body;
    private final LocalDateTime dateCreated;

    public DiaryEntry(int id, String title, String body) {
        this.id = id;
        this.title = title;
        this.body = body;
        this.dateCreated = LocalDateTime.now();
    }

    public int getId() {
        return id;
    }
    public void setTitle(String title) {
        this.title = title;
    }
    public String getTitle() {
        return title;
    }
    public void setBody(String body) {
        this.body = body;
    }
    public String getBody() {
        return body;
    }
    public String getDateCreated() {
        return dateCreated.toString();
    }
}

File: GATE ONE/DAYSEVEN/PYTHON/DIARY/DiaryEntry.py
from datetime import datetime

class DiaryEntry:
    def __init__(self, id, title, body):
        self.id = id
        self.title = title
        self.body = body
        self.dateCreated = datetime.now()

    def getId(self):
        return self.id

    def setTitle(self, title):
        self.title = title

    def getTitle(self):
        return self.title

    def setBody(self, body):
        self.body = body

    def getBody(self):
        return self.body

    def getDateCreated(self):
        return self.dateCreated.strftime("%Y-%m-%d %H:%M")